ios/A0Auth0.m

```diff
@@ -39,6 +41,37 @@ - (dispatch_queue_t)methodQueue
     [self terminateWithError:nil dismissing:YES animated:YES];
 }
 
+RCT_EXPORT_METHOD(hasValidCredentialManagerInstance:(RCTPromiseResolveBlock)resolve
+                  rejecter:(RCTPromiseRejectBlock)reject) {
+    BOOL valid = [self checkHasValidCredentialManagerInstance];
+    resolve(@(valid));
+}
+
+
+RCT_EXPORT_METHOD(initializeCredentialManager:(NSString *)clientId domain:(NSString *)domain) {
+    [self tryAndInitializeCredentialManager:clientId domain:domain];
+}
+
+RCT_EXPORT_METHOD(saveCredentials:(NSDictionary *)credentials resolver:(RCTPromiseResolveBlock)resolve rejecter:(RCTPromiseRejectBlock)reject) {
+    [self.credentialsManagerBridge saveCredentialsWithCredentialsMap:credentials resolve:resolve reject:reject];
```

Comment:
This was renamed to credentialsDict. Are we building the app as part of the CI job to make sure the native code compiles?

Reply:
@Widcket we have added native builds as a part of another ticket where we add sample app to the project (SDK-3598)